contracts/0.4.24/Lido.sol

```diff
@@ -458,14 +461,33 @@ contract Lido is StETHPermit, AragonApp {
     }
 
     /**
-     * @dev Sets limit on amount of ETH to withdraw from execution layer rewards vault per LidoOracle report
-     * @param _limitPoints limit in basis points to amount of ETH to withdraw per LidoOracle report
+     * @dev Set max positive rebase allowed per single oracle report
+     * token rebase happens on total supply adjustment,
+     * huge positive rebase can incur oracle report sandwitching.
+     *
+     * stETH balance for the `account` defined as:
+     * balanceOf(account) = shares[account] * totalPooledEther / totalShares = shares[account] * shareRate
+     *
+     * Suppose shareRate changes when oracle reports (see `handleOracleReport`)
+     * which means that token rebase happens:
+     *
+     * preShareRate = preTotalPooledEther() / preTotalShares()
+     * postShareRate = postTotalPooledEther() / postTotalShares()
+     * R = (postShareRate - preShareRate) / preShareRate
+     *
+     * R > 0 corresponds to the relative positive rebase value (i.e., instant APR)
+     *
+     * NB: The value is not set by default (explicit initialization required),
+     * the recommended sane values are from 0.05% to 0.1%.
+     *
+     * @param _maxTokenPositiveRebase max positive token rebase value with 1e9 precision:
+     *   e.g.: 1e6 - 0.1%; 1e9 - 100%
+     * - passing zero value is prohibited
+     * - to allow unlimited rebases, pass max uint256, i.e.: uint256(-1)
```

Comment:
```suggestion
     * - to allow unlimited rebases, pass max uint256, i.e.: type(uint256).max
```

Reply:
it's not available for 0.4, though readability is better, ty :+1: